docs/topics/mta-cli-run-single-app.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="mta-cli-run-single-app_{context}"]
+= Running the {ProductShortName} {CLINameTitle} against an application
+
+You can run the {ProductFullName} {CLINameTitle} against an application.
+
+.Procedure
+
+. Open a terminal and navigate to the `<{ProductShortName}_HOME>/` directory.
+
+. Run the `{mta-cli}` script, or `{mta-cli}.exe` for Windows, and specify the appropriate arguments:
+
++
+[source,terminal,subs="attributes+"]
+----
+$ ./{mta-cli} analyze --input <path_to_input> \
+    --output <path_to_output> --source <source_name> --target <target_source> \
+----
++
+* `--input`: The application to be evaluated.
+* `--output`: The output directory for the generated reports.
+* `--source`: The source technology for the application migration. For example, `weblogic`.
+* `--target`: The target technology for the application migration. For example, `eap8`.
+
+. Access the report.
+
+[id="command-examples_{context}"]
+== {ProductShortName} command examples
+
+[discrete]
+=== Running {ProductShortName} on an application archive
+
+The following command analyzes the link:https://github.com/windup/windup/blob/master/test-files/jee-example-app-1.0.0.ear[jee-example-app-1.0.0.ear] example EAR archive for migrating from JBoss EAP 5 to JBoss EAP 7:
```

Comment:
Why do we need to provide links to demo apps from the Windup repository? I don't see the point of this, and after September 30th that repo is likely to be archived.